pubspec.yaml

```diff
@@ -21,11 +21,11 @@ dependencies:
   yaml: ^3.1.0
 
 dev_dependencies:
-  build_runner: ^2.0.0
-  json_serializable: ^4.1.0
-  test: ^1.15.7
+  build_runner: ^2.1.2
+  json_serializable: ^5.0.2
```

Comment:
same here. minimum needed for null safety is 4.1.0. Is that so we can get the right build_config ^1.0 version?

Reply:
https://github.com/Workiva/dependency_validator/pull/92#discussion_r915272504